core/util/src/main/scala/net/liftweb/util/HttpHelpers.scala

```diff
@@ -138,7 +138,7 @@ trait HttpHelpers {
   */
   def toHashMap[A,B](in : Map[A,B]) : HashMap[A,B] = {
     val ret = new HashMap[A,B];
-    in.keysIterator.foreach { k => ret += Pair(k, in(k)) }
+    in.keysIterator.foreach { k => ret += ((k, in(k))) }
```

Comment:
Can this be `(k -> in(k))`? The double and triple parens are kind of blowing my mind hehe.